packages/odyssey-react/src/components/Table/TableContainer.tsx

```diff
@@ -26,11 +35,11 @@ export type Props = {
    * The visible heading for the table
    */
   title: ReactNode;
-} & ComponentProps<"figure">;
+}
 
 type Ref = HTMLElement;
```

Comment:
is this type alias required or used elsewhere? Might be nice to just pass `HTMLElement` in directly on line 42 here